e2e/specs/attachments.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, type Patient, deletePatient } from '../commands';
+import { test } from '../core';
+import { AttachmentsPage } from '../pages';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('Add and remove an attachment', async ({ page }) => {
+  const attachmentsPage = new AttachmentsPage(page);
+  const filePath = './e2e/support/upload/brainScan.jpeg';
+
+  await test.step('When I go to the Attachments page', async () => {
+    await attachmentsPage.goTo(patient.uuid);
+  });
+
+  await test.step('And I click on the `Record attachments` link', async () => {
+    await page.getByRole('button', { name: /record attachments/i }).click();
+  });
+
+  await test.step('Then I should see the `Add attachment` model launch in the workspace', async () => {
+    await expect(page.getByText(/add attachment/i)).toBeVisible();
+  });
+
+  await test.step('When I click on the `Upload files` tab', async () => {
+    await page.getByRole('tab', { name: /upload files/i }).click();
+  });
+
+  await test.step('And I choose the attachment file to upload', async () => {
```

Comment:
I'm thinking otherwise. Why?  `And I upload an image` ideally suggests that all the actions for uploading the file are covered within this step. However, the action of adding an image is in the next step so this step is just selecting the file to upload.

Reply:
Makes sense!